tests/unit/store/unload-test.js

```diff
@@ -47,7 +49,7 @@ testInDebug("unload a dirty record asserts", function(assert) {
       }
     });
 
-    store.findRecord('record', 1).then(function(record) {
+    store.findRecord('record', 1, { backgroundReload: false }).then(function(record) {
```

Comment:
The `store.findRecord` is unnecessary for this test. Let's use `store.peekRecord` instead.